### ISSUE TITLE
image shape

### ISSUE THREAD

**Issue by wtskk** [noagarcia/keras_rmac#7]
x = image.img_to_array(img) #把图像转换为数组类形numpy.ndarray
 x = np.expand_dims(x, axis=0)
 x = utils.preprocess_image(x)
The x output shape here is (1, 683, 1024, 3)
but x = utils.preprocess_image(x)
    x[:, 0, :, :] -= 103.939
    x[:, 1, :, :] -= 116.779
    x[:, 2, :, :] -= 123.68
The second dimension is the number of channels, but the last dimension of the shape of x is the number of channels

**Comment by noagarcia** [comment]
Which backend are you using? Please, be aware the dimensions may be different

**Comment by wtskk** [comment]
tensorflow

**Comment by noagarcia** [comment]
Yeah, this repository works with Theano backend. You will need to slightly modify the code if you want to use the Tensorflow backend.
